README.md

```diff
@@ -26,38 +26,150 @@ guarantee these changes will be backwards-compatible.
   - [User Stories](https://docs.google.com/document/d/1wT8e8wBHMrSRHY4UF_60GCgyWGqvYye4THvaDARPySs/edit)
   - [Features](https://docs.google.com/document/d/1JNP6DgSK-c6KqveIhQk-n_HAw3hsZkL-okoleM43NgA/edit)
   - [Design Space](https://docs.google.com/document/d/1TpQlQYovCoX9FkpgsoxzdvZplghudHAiQOame30A-v8/edit)
-- [Frequently Asked Questions](FAQ.md)
+- [Frequently Asked Questions](docs/FAQ.md)
 
-## Usage
+## Setup
 
 Get the tool via
 
 ```sh
 $ go get -u github.com/golang/dep/cmd/dep
 ```
 
-Typical usage on a new repo might be
+To start managing dependencies using dep, run the following from your project root directory:
 
 ```sh
 $ dep init
-$ dep ensure -update
 ```
 
-To update a dependency to a new version, you might run
+This does the following:
+
+1. Look for [existing dependency management files](docs/FAQ.md#what-external-tools-are-supported) to convert
+1. Check if your dependencies use dep
```

Comment:
I don't want to sidetrack/bikeshed this issue (so we can continue discussing in a new issue if necessary), but would ask that we don't backtick _every_ mention of dep. 😀

One easy way to think about it this: only use backticks if we are telling them to run a command, referring to a dep command, etc. Anytime we are simply referring to the dep project, leave off the special formatting.

Examples:

* Check if your dependencies use dep...
* Run `dep ensure` to populate the vendor directory...
* We commend using `dep` in your Makefiles...
* Verify that `dep` is on your $PATH...